func_test.go

```diff
@@ -86,3 +87,38 @@ func Test_qsort(t *testing.T) {
 		}
 	}
 }
+
+func TestRegisterFunc_Floats(t *testing.T) {
+	library, err := getSystemLibrary()
+	if err != nil {
+		t.Errorf("couldn't get system library: %s", err)
```

Comment:
t.Fatalf

Reply:
Don't we want to run both functions before failing? t.Fatalf ends the test so we don't know if the other one succeeded or not.

Reply:
When a system library is not found, we cannot test the other tests, right?

Reply:
Or, is it possible that a library is found and an error is returned at the same time...?

Reply:
That is true bc it will panic if the symbol isn't found. However, if the symbol is found it may not return the correct value which is what is being tested. We'd want to test both float32 and float64 even if one fails.

Reply:
Does `getSystemLibrary` search symbols? I don't suggest replacing all `Errorf` with `Fatalf`. Rather, I suggest replacing only two `Errorf` for `getSystemLibrary` and `openLibrary`.

Reply:
Oh ok. That makes sense. Should we do that then for `Test_qsort` too?

Reply:
Yeah. (Also TestRegisterFunc?)